<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>423</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Send Message</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="options_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>To:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="options_combo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="message_edit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>281</width>
      <height>231</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter message or drag and drop files here, use ctlr+enter to send message</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>281</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="resizeMode">
     <enum>QListView::Fixed</enum>
    </property>
   </widget>
   <widget class="QLabel" name="pathLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Files Attached:</string>
    </property>
   </widget>
  </widget>
  <widget class="menui" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionInsert_File"/>
    <addaction name="actionInsert_directory"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionInsert_File">
   <property name="text">
    <string>Insert File</string>
   </property>
  </action>
  <action name="actionInsert_directory">
   <property name="text">
    <string>Insert directory</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>menui</class>
   <extends>QMenuBar</extends>
   <header>menui.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
